cumulus/client/cli/Cargo.toml

```diff
@@ -17,10 +17,10 @@ codec = { workspace = true, default-features = true }
 url = { workspace = true }
 
 # Substrate
```

Comment:
Not for this PR maybe, but I'm also not a huge fan of dependencies being separated with comments like this. If we're sorting them now I think it's a good time to remove them

Reply:
I also thought about this before. I agree you. Since they have already been "prefixed". It's easy to distinguish.